docs/troubleshooting/no-space-left-on-device.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_position: 2
+---
+
+# No space left on device
+
+If you get an error in the deploy log that says `No space left on device`, here is how to fix.
+
+## Error message
+
+```code
+fatal: cannot create directory at 'some_large_file.txt': No space left on device
+2021-12-06 10:49:49: [CYCLIC] Build Failed
+Push a new commit to start another.
+```
+
+## Solution
+
+The only resolution at this time is to trim your dependencies or devDependencies.
+
+## Why does this happen?
+
+Cyclic builds and deploys apps using serverless technologies. AWS provides only 256MB of disk space for each lambda function.
```

Comment:
lambda /tmp space is limited to 512
code at runtime /var/task is limited to 256 - whatever we are using for the runtime layer, which is < 6mb so like 250 for whatever is produced in /dist